main.py

```diff
@@ -1,7 +1,40 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre
+from db.models import Actor
 
 
-def main() -> QuerySet:
-    pass
+def main():
+    # Create genres
+    western = Genre.objects.create(name="Western")
+    action = Genre.objects.create(name="Action")
+    drama = Genre.objects.create(name="Dramma")
```

Comment:
There is a typo in the genre name 'Dramma'. It should be 'Drama' to maintain consistency with the update operation on line 23.